hazelcast-integration/hibernate-2ndlevel-cache/README.md

```diff
@@ -4,60 +4,81 @@ In this repository, you can find a sample implementation of hibernate 2nd level
 
 ## Prerequisites
 
-You should have installed Apache Maven(http://maven.apache.org/download.cgi).
+You should have installed Apache Maven and Mysql database.
```

Comment:
> and Mysql database

We could probably simplify this step by preparing a set of Docker commands. Let's keep that in mind for the next iteration

Reply:
That's exactly what I mean by `glitches`. h2 db, for instance, would simply serve the purpose here without dealing any Mysql stuff. I will take care of them in the next iteration.